spec/test_data/multimedia/file_manifest.csv

```diff
@@ -7,7 +7,7 @@ aa111aa1111,aa111aa1111_001_b_pm.wav,"Tape 1, Side B",2,yes,yes,yes,file,
 aa111aa1111,aa111aa1111_001_b_sh.wav,,,no,no,no,,
 aa111aa1111,aa111aa1111_001_b_sl.mp3,,,yes,yes,yes,,
 aa111aa1111,aa111aa1111_001_img_2.jpg,,,yes,yes,yes,,
-aa111aa1111,aa111aa1111.pdf,Transcript,3,yes,yes,yes,text,
+aa111aa1111,aa111aa1111.pdf,Transcript,3,yes,yes,yes,file,
```

Comment:
If this is changing, in that there will need to be some user education.  Unless the users no longer do it this way?  (You could check Geoff Willard's jobs in prod).   Andrew would need to be alerted.

Reply:
I think this is bad test data.  "text" is not a valid resource type. See https://github.com/sul-dlss/cocina-models/blob/main/lib/cocina/models/file_set_type.rb#L7-L22

Reply:
ah!  thanks.